components/crypto_exchange/browser/crypto_exchange_region_util.cc

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <string>
+#include <vector>
+
+#include "brave/components/crypto_exchange/browser/crypto_exchange_region_util.h"
+
+#include "components/prefs/pref_service.h"
+#include "components/country_codes/country_codes.h"
+
+namespace crypto_exchange {
+
+bool IsRegionSupported(PrefService* pref_service,
+    std::vector<std::string> regions,
```

Comment:
This vector should probably be a const reference.

Reply:
Ya: 
```
bool IsRegionSupported(PrefService* pref_service,
    const std::vector<std::string> &regions,
```

Reply:
done